include/sparrow/data_type.hpp

```diff
@@ -103,7 +120,8 @@ namespace sparrow
         float32_t,
         float64_t,
         std::string,
-        std::vector<byte_t>
+        std::vector<byte_t>,
+        sparrow::timestamp
```

Comment:
I think there is one type missing before that one, see compare with the order of the enum, `BINARY` matches `std::vector<byte_t>` but we dont have something for `FIXED_SIZE_BINARY`. Not sure which type is missing though. @JohanMabille any idea what it would match?

Reply:
I think `FIXED_SIZE_BINARY` is meant to be used for a fixed size binary type which is not a primitive type, in which case we could use a `struct` as a type to test `FIXED_SIZE_BINARY`.

Reply:
I took the values from [Arrow implementation](https://github.com/apache/arrow/blob/main/cpp/src/arrow/type_fwd.h), not sure which type this value should match.

Reply:
Also, must we work on supporting [`DATE32` and `DATE64` which come before `TIMESTAMP`](https://github.com/apache/arrow/blame/b2e8c33c86c819b167a1cbca834da3c9047a9350/cpp/src/arrow/type_fwd.h#L358-L364)?

Reply:
The support for DATE32 and DATE64 can be added later. Explicit integer identifiers would be required later for backward compatibility, but it doe snot matter for now, nor the enum "stability".